### DISCUSSION TITLE
How to use OXLint with webpack?

### DISCUSSION THREAD

**Discussion by cjnoname** [oxc-project/oxc#7550] [Feature Request]
Is there a plugin available for this?

**Comment by cjnoname** [comment]
We are using Rspack too.

**Comment by pumano** [comment]
for using oxlint @cjnoname you can just install oxlint via your favorite package manager and use it via command line. No need to use it via webpack or something else. It's like eslint tool

**Reply by cjnoname** [reply]
> for using oxlint @cjnoname you can just install oxlint via your favorite package manager and use it via command line. No need to use it via webpack or something else. It's like eslint tool

Mate, we use eslint-webpack-plugin for local development with both Webpack and Rspack. It would be beneficial for Oxlint to support this plugin, ensuring compatibility across Webpack and Rspack.

**Comment by Boshen** [comment]
No plugin available at present.

**Reply by cjnoname** [reply]
Is there a way to make Oxlint work together with Rspack, similar to how the eslint-webpack plugin works?

**Comment by asilgag** [comment]
Please check https://www.npmjs.com/package/oxlint-webpack-plugin (I'm the author of that package)
